sonar-plugin-api/src/main/java/org/sonar/api/server/ws/WebService.java

```diff
@@ -94,9 +94,12 @@
  * provided by {@link RequestHandler#handle(Request, Response)}.
  *
  * @since 4.2
+ *
+ * @deprecated since 6.5 use {@link WebServiceDefinition}
  */
 @ServerSide
 @ExtensionPoint
+@Deprecated
 public interface WebService extends Definable<WebService.Context> {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)